src/DropdownMenu/DropdownText.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled, { CSSObject } from "styled-components";
+import { Text } from "../Type";
+import { TextProps } from "../Type/Text";
+
+const DropdownText: React.FC<TextProps> = styled(Text)(
+  ({ theme }: TextProps): CSSObject => ({
+    color: theme.colors.darkGrey,
+    display: "block",
+    width: "100%",
+    border: "none",
+    textAlign: "left",
+    backgroundColor: "transparent",
+    transition: ".2s",
```

Comment:
It's a good idea to specify the transition properties here, i.e what we want to animate. If none are specified it's `all` by default.

we can define it using `transition: <property> <duration>`

Reply:
@haideralsh I did this by looking at what was in the `DropdownButton`, `DropdownLink`, and `DropdownItem` and made it match those. I think if I change this, they should all change. Also, I'm unsure exactly what transitions are being animated.

Reply:
Ok I see. we can leave it the way it is.